.gitignore

```diff
@@ -23,13 +23,15 @@ wikiconfig_*.py
 *.pyo
 *.orig
 *.rej
+# files created by quickinstall.py
 activate.bat
 deactivate.bat
 moin.bat
 m.bat
 activate
-moin
 m
+# files created by make.py
 m-*.txt
+upload.py
```

Comment:
`upload.py` is the rietveld (google codereview) uploader. I removed related code and guess we we won't use that again now we can do code reviews on github PRs.